tests/test_dragonnet.py

```diff
@@ -3,21 +3,21 @@
 except ImportError:
     pass
 from causalml.dataset.regression import simulate_nuisance_and_easy_treatment
-import shutil
 import pytest
 
 
 @pytest.mark.tf
-def test_save_load_dragonnet():
+def test_save_load_dragonnet(tmp_path):
     y, X, w, tau, b, e = simulate_nuisance_and_easy_treatment(n=1000)
 
     dragon = DragonNet(neurons_per_layer=200, targeted_reg=True, verbose=False)
     dragon_ite = dragon.fit_predict(X, w, y, return_components=False)
     dragon_ate = dragon_ite.mean()
-    dragon.save("smaug")
+
+    model_file = tmp_path / "smaug.h5"
```

Comment:
@jeongyoonlee did you mean `f"{tmp_path}/smaug.h5"` ? I'm not super familiar with the code here

Reply:
tmp_path is a Pathlib object and this is a valid syntax. :)

Reply:
i see, makes sense then!